### PR TITLE
centered signup form

### PR DESCRIPTION
**Pull request by Pybite**
resolved issue #28

### CODE REVIEW

File: src/signup.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 body {
   font-family: "Montserrat", sans-serif;
+  display: flex;
```
Comment (nbkhope) [inline src/signup.css:3]: We usually don't want to target the body as a whole. You should add a centered container class to the main content element instead.

Comment (Pybite) [reply src/signup.css:3]: ok, will work on it

Comment (nbkhope) [reply src/signup.css:3]: If you choose to do the way you're doing with flex, you will need a wrapper like this:

```
<main class="centered-container"> 
  <div> <--- you need this inner wrapper
    main content goes here
  </div>
</main>
```

Comment (Pybite) [reply src/signup.css:3]: I've tried doing as such but it breaks the nav bar

Comment (Pybite) [reply src/signup.css:3]: fixed !

**Review by nbkhope — APPROVED** [review]

**Pull request merged**